dojo/utils.py

```diff
@@ -2108,3 +2121,102 @@ def is_scan_file_too_large(scan_file):
         if size > settings.SCAN_FILE_MAX_SIZE:
             return True
     return False
+
+
+def sla_compute_and_notify(*args, **kwargs):
+    """
+    The SLA computation and notification will be disabled if the user opts out
+    of the Findings SLA on the System Settings page.
+
+    Notifications are managed the usual way, so you'd have to opt-in.
+    Exception is for JIRA issues, which would get a comment anyways.
+    """
+    def _notify(finding, title):
+        create_notification(
+            event='sla_breach',
+            title=title,
+            finding=finding,
+            sla_age=sla_age
+        )
```

Comment:
just thinking out loud: if a user once was an authorized user for a product and has configured notifications, the user will still receive those even after it would have been removed as authorized user from the product. probably not wanted/needed to solve here, so #2700